src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  users: User[];
+  onAdd: (todo: Omit<Todo, 'id'>) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ users, onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [titleHasError, setTitleHasError] = useState(false);
+
+  const [userSelect, setUserSelect] = useState(0);
+  const [userSelectHasError, setUserSelectHasError] = useState(false);
+
+  const isAllowedSymbols = (input: string): boolean => {
+    const pattern = /^[a-zA-Z А-ЩЬЮЯҐЄІЇа-щьюяґєії0-9]+$/;
```

Comment:
The regular expression used in the 'isAllowedSymbols' function only accepts Cyrillic and Latin characters. If the application is intended to support more languages, this could be a problem. Consider creating a more inclusive regular expression or handle this validation differently.